monai/bundle/scripts.py

```diff
@@ -469,11 +469,33 @@ def get_bundle_info(
     return bundle_info[version]
 
 
+def patch_bundle_handler(parser: ConfigParser, settings: dict):
```

Comment:
Hope this is for adding any tracking related features, not necessarily any handlers, so the name could be "patch_bundle_tracking"?

Reply:
Sure, updated function name and doc-string in the latest commit.

Thanks.